### ISSUE TITLE
Accessibility broken for VGS InputFields

### ISSUE THREAD

**Issue by mikkoville** [verygoodsecurity/vgs-collect-android#801]
**Describe the bug**
If user has Talkback enabled and clicks on a vgs InputFields like:
`VGSCardNumberEditText`, `CardVerificationCodeEditText` or `ExpirationDateEditText` the screen reader reads aloud the `hint` label set to the field with `.setHint()` and not the `contentDescription` set by` setContentDescription()`.

This is bad since hint is often used for something visual like in the case of Expiry the format MM/YY. This is not descriptive for users who use talkback.

Instead for Talkback the contentDescription field should be read so more descriptive text is placed there

**To Reproduce**
1. Set hint for VGSCardNumberEditText or other mentioned InputFields
2. Set contentDescription for the same field
3. Enable Talkback
4. See that hint is read by Talkback instead of the contentDescription

**Expected behavior**
ContentDescription should be respected by ScreenReaders

**Environment (please complete the following information):**
 - SDK Version 1.9.13

**Comment by VadymBsv** [comment]
@mikkoville I will check it

**Comment by VadymBsv** [comment]
@mikkoville Confirm please 
`VGSCardNumberEditText, CardVerificationCodeEditText or ExpirationDateEditText`
Do you use this elements by themselves or do you cover them with VGSTextInputLayout for example?
Because here might be a problem when you have an EditText wrapped by a TextInputLayout, the screen reader will typically read only the hint set on the TextInputLayout and ignore the contentDescription set on the EditText.
So you just need to move your content description upper in hierarchy

**Comment by mikkoville** [comment]
Hi. I am actually recently been using the compose version `VGSCardNumberEditTextWrapper` that uses the `VGSTextInputLayout` under the hood. If I set contentDescription for the layout it is read out aloud by Talkback but it is read separately of the hint. So first it reads the contentDescription and then I need to swipe again for it to read the hint.

It seems I need to use only one of these hint or contentDescription but not both. Otherwise the experience is not good. Too bad since hint would be better as the visual hint and contentDescription could be used as the alternative one for users with Talkback 🤔

**Comment by DmytroDm** [comment]
Hi @mikkoville, are you still experiencing this issue on the latest version of VGS Collect?

If you're using Jetpack Compose, we'd also recommend upgrading to VGS Collect 2.0, which introduces fully native Compose widgets with improved accessibility support and a more idiomatic Compose API.

If the issue is still reproducible on the latest version, please let us know which version you're using and we'll be happy to investigate further.
